### DISCUSSION TITLE
"Reinventing cosmology: New research puts age of universe at 26.7 - not 13.7 - billion years" using tired-light

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#102] [Redshift Models]
"Our universe could be twice as old as current estimates, according to a new study that challenges the dominant cosmological model and sheds new light on the so-called 'impossible early galaxy problem.' "

🔗 [https://www.sciencedaily.com/releases/2023/07/230711133118.htm](https://www.sciencedaily.com/releases/2023/07/230711133118.htm)

**Comment by HanDeBruijn** [comment]
_26.7 -- not 13.7 -- billion years_

That's actually an old (1992) statement: see formula (13) in [FLAT SPACETIME COSMOLOGY](http://www.alternatievewiskunde.nl/cosmology/narlikar.pdf) by Jayant Narlikar and Halton Arp, where it reads

$$
\large H_0 = \frac{2}{t_0}
$$

This result is also mentioned in ["Big Bang: Tweak it or throw it out? "](https://github.com/orgs/a-cosmology-group/discussions/39#discussioncomment-6156371): _we easily find that_ $T_0-A=2/H$, _meaning that the current "age of the universe" is_ $2/H$, _not_ $1/H$ _as according to many other models._ Giving 27.4 -- not 26.7 -- billion years (minor correction).

### Update

_Zwicky's tired light theory proposes that the redshift of light from distant galaxies is due to the gradual loss of energy by photons over vast cosmic distances. [ .. ] Gupta found that "by allowing this theory to coexist with the expanding universe, it becomes possible to reinterpret the redshift as a hybrid phenomenon, rather than purely due to expansion."_

Stop talking! Calculate! Let the expanding universe redshift be $z_1$ and the tired light redshift be $z_2$ . Then the total redshift $z$ is

$$
1+z=(1+z_1)(1+z_2)=e^{H_1(t_0-t)}e^{H_2(t_0-t)}=e^{(H_1+H_2)(t_0-t)}=e^{H(t_0-t)} \quad \Longrightarrow \quad H=H_1+H_2
$$

where $H_1=$ expanding universe Hubble constant, $H_2=$ tired light Hubble constant and $H=$ hybrid Hubble constant; $(t_0-t)$ is how long ago the redshifted radiation was emitted. If the tired light contribution is the same as the expanding universe contribution then we have $H=H_1+H_2=2H_1$ and so

$$
\frac{1}{\mbox{age}} = H_1 = \frac{H}{2} \quad \Longrightarrow \quad \mbox{age} = \frac{2}{H}
$$

Which is the same result as obtained with our [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs).

Gupta proceeds with _the idea of evolving "coupling constants," as hypothesized by Paul Dirac_, thereby making his theory even more (and IMHO unnecessary) complicated.

**Comment by khuramonline** [comment]
“Age” was based on CMB “data” and the assumption that CMB is the proof of expanding universe. Now if “age” is almost double then CMB is no more the proof of expanding universe.

**Reply by RedshiftDrift** [reply]
Agreed, the author "shoots himself in the foot".  The paper is ressurecting an old (and dead) Eddington-Lemaître model which doesn't agree with other observations, with the addition of more free parameters and fewer supporting arguments for the CMB.

**Reply by khuramonline** [reply]
So either they will set new distances to all the galaxies through an adjustment in formula. (as if galaxies choose their distances according to their formulas).

Or ... There will be a dark age of about 13 billion years. When they will accept that mess is mess?

**Reply by budrap00** [reply]
`When they will accept that mess is mess?`

The nonsense will only stop when Science Departments stop funding it, not before.

**Reply by khuramonline** [reply]
If they accept that matters (i.e. universe expanding or not or dark matter is a reality or not) can be sorted out using simple logic then there is no scope of funding. Need of funding comes when they project that matters require specially qualified genius coupled with the need for expensive instruments and experiments.

**Comment by LifeIsStrange** [comment]
> . While tired light (TL) models have been shown to comply with the JWST angular galaxy size data, they cannot satisfactorily explain isotropy of the cosmic microwave background (CMB) observations or fit the supernovae distance modulus versus redshift data well. We have developed hybrid models that include the tired light concept in the expanding universe.

Firstly the idea that tired light models do not fit the Hubble diagram is IMO wrong, is there something I'm missing out? He did not give a single source despite being the reason of existence of his model.
The second reason given is about the CMB, which actually some tired light mechanisms aims to reproduce as specific reradiation, but mostly the most simple explanation is that the low end of the spectrum is dominated by synchrotrons via magnetospheres (very few people know that the 1960s discovery of the CMB (4Ghz) is in fact consensually synchrotron dominated and therefore refuted) and for all the spectrum, generated by cosmic dust as proven here
https://iopscience.iop.org/article/10.1088/1742-6596/2197/1/012026/meta
@RedshiftDrift This paper is one of the best explanation of the CMB unfortunately the calculated size range and density of cosmic dust is sensitive to the electron density (calculated as 0.5 el/m3), it would be nice if you could re-run his calculations with the new estimate you gave in your fast radio burst preprint, of 0.41 el/m3, as the previous cosmic dust range has likely been falsified.

In summary Gupta paper doesn't provide convincing added value over TL besides complexity, moreover it can be clearly seen from his graph that his curve seems to be a poorer fit of the angular distance test than TL alone.
However I find it absolutely remarkable that this is the first expanding model of the universe that simultaneously take into account tired light (which after all is a truism to some extent considering inverse compton and the solar excess redshift), hybrid models seemed like a basic idea to me even when I was just beginning at studying cosmology, the fact we had to wait for 123 years before this idea being proposed is a prime exemple of the cognitive biases and sociology of cosmology, the idea is not original, but it couldn't be conceived because of the tribalism between the two ideological groups/dichotomy, truly it is quite pathetic for this entire field.
Other hybrids that are reasonable are for example, the idea that the universe had a hot start, but at a fixed or infinite size and never expanded. This possibility is too often ignored by static cosmologists despite being the only alternative (along with cold start) to the paradoxal eternal, without start universe, and being able in theory (haven't done the math) to explain the CMB as the line of last scattering in a static universe.

**Reply by ExpEarth** [reply]
@LifeIsStrange 

I agree that Gupta's paper is not really convincing and also confusing.  He also doesn't account for the CMB temperature problems or the time dilation problem in TL models.  But I am amazed too by how much attention he got with this weird hybrid.  From the videos I saw of him, he also was not expecting so much attention.

What it really shows is two things.  First, the LCDM model is fading fast and people are seriously looking for alternatives.  At the same time, Gupta may be very wise in going this hybrid route.  It allows ideas like TL to enter through the back door, so to speak.  I have been forced to follow a similar route in my paper, which has been stuck in journal limbo for almost a year now.  We have to face it that most people - and almost all reviewers - seriously think the universe is expanding.  A straight on rejection of this cannot be easily published.  The choice is between not getting your article published at all or trying to write it in such a way that open-minded people can perhaps see something in it.

**Reply by LifeIsStrange** [reply]
I agree that Gupta might potentially be a pragmaticist/cynic. As for popularity, it is not everyday that an alternative _expanding_ model is published, even more so an hybrid one and with a specific "surprising" age.
The impossibly early galaxies/quasars problem has become untenable (see the recent post on a cosmology group about spectroscopically confirmed large and luminous galaxies at Z=14.32 (new record, 290 millions years old) while a paper recently published about galaxies at "only" z=8 was already talking about those galaxies being statistically born before the start of the universe (by 400 to 1 billion years)).
Gupta theory is the only expanding theory that I know that is compatible with a much older universe and it likely be one of the last refuges.
Side note: actually the cosmological tests with JWST paper already combined TL with LCDM right before Gupta's paper which reuse its data so he might have taken inspiration, however the LCDM + TL for some reason while better than LCDM, still gives a wrong tolman curve contrary to CCC.

> the LCDM model is fading fast

Sadly even if it will become even more painfully obviously non credible, there is always the option of reality denial for LCDM believers. After all, LCDM is already completely falsified with certainty.
https://iopscience.iop.org/article/10.3847/2041-8213/acdb49

> time dilation problem in TL models.

There is no cosmic time dilation problem in tired light, it is hard to say something more distant from the truth.
cosmic time dilation is empirically falsified 
https://arxiv.org/abs/0901.4169
https://arxiv.org/abs/1711.11237
https://arxiv.org/pdf/1004.1824
100% of studies shows no signal in gamma ray bursts
Studies showing time dilation in supernovae is scientific fraud and an artefact of the SALT2 algorithm, as has been proven
Only one study "bayesian analysis" shows against all others, time dilation for quasars (and only when bined IIRC) and this study conflate quasar variability dependence on redshift as a true dilation signal versus evolution or intrinsic redshift or systematics like malmquist bias.
Note that remaining probes that nobody has tested (maybe you could?) would be ultracompact radio sources and fast radio bursts. Also since Miras have gold standard variability they would be an excellent probe?

> The choice is between not getting your article published at all or trying to write it in such a way that open-minded people can perhaps see something in it.

This is a tragedy. I sometimes wish for a Roko's basilisk..

BTW is your name from the expanding earth theory?

**Reply by ExpEarth** [reply]
On the supernova time dilation, I tried to read Crawford's papers refuting it long ago and could not make much sense of them.  My general approach is if the cosmological or physical 'cure' looks more complicated than the 'illness', it could be time to move on.  At this point in time I don't have any reason to doubt the supernova data or the quasar data.  (I say this in part because I can explain it with my own model, as well as the CMB temperature problem.)

`BTW is your name from the expanding earth theory?`

Yes it is!

https://www.sciencedirect.com/science/article/pii/S1674984718301678
https://hgss.copernicus.org/articles/7/125/2016/

**Reply by RedshiftDrift** [reply]
@LifeIsStrange 
> ... about the CMB, which actually some tired light mechanisms aims to reproduce as specific reradiation, but mostly the most simple explanation is that the low end of the spectrum is dominated by synchrotrons via magnetospheres [....] and for all the spectrum, generated by cosmic dust as proven here
[https://iopscience.iop.org/article/10.1088/1742-6596/2197/1/012026/meta](https://iopscience.iop.org/article/10.1088/1742-6596/2197/1/012026/meta)
@RedshiftDrift This paper is one of the best explanation of the CMB unfortunately the calculated size range and density of cosmic dust is sensitive to the electron density (calculated as 0.5 el/m3), it would be nice if you could re-run his calculations with the new estimate you gave in your fast radio burst preprint, of 0.41 el/m3, as the previous cosmic dust range has likely been falsified.

The value $0.5 \mbox{ e/m}^3$ is not given an uncertainty in Trinchera's paper.  Like every physical measurement, this value has an uncertainty which I estimate roughly as $0.5^{+2.0}_{-0.4} \mbox{ e/m}^3$.  The uncertainty is mainly produced by the non-homogeneity of the IGM near galaxies, which in turn produce a variable tired-light redshift and variations on the dust number-density.  Even when averaged over long distances, the variations in the redshift produce large uncertainties on the estimated values of the Hubble constant (and all the anisotropies studied by Subir Sarkar).

There is not much point re-running Trinchera's calculation with other values, it will just change the amount of dust needed to convert the radiation into a blackbody spectrum at 2.7 K.  My 2017 preprint gave $0.41 \mbox{ e/m}^3$ with one FRB measurement, but a more recent calculation [^1] using 12 FRBs gave $0.25 \mbox{ e/m}^3$ (probably with an uncertainty like $^{+1.0}_{-0.15} \mbox{ e/m}^3$ ) but shows correlations between different FRBs that are a result of our incomplete knowledge of the Dispersion Measure in the Milky Way.  Now 50 FRBs have been associated with a host galaxy, but I don't have the time to redo the calculation.

Dust models of the CMB have been proposed a long time ago (dust grains, carbon or metallic whiskers [F. Hoyle *et al.*], etc.).  There are two major objections to those:
1) Dust grains need to have a very specific size-distribution to produce a blackbody spectrum.  Trinchera avoids mentioning this in his paper.  There is no way to explain how dust grains would be formed with the exact size-distribution required to produce the CMB at 2.7 K, the required fine-tuning cannot happen naturally.
2) Dust produces absorption, especially at shorter wavelengths.  This absorption could be observed as a reddening of the spectrum (not "redshifting", but "reddening" like the red sunset).  On the other hand, spectra of supernovae are extremely well characterized up to very large redshifts, and very little reddening is observed [Adam Riess' papers].  *Intergalactic space is extremely transparent.*  This is incompatible with the required absorption needed to produce blackbody radiation.
(Eric Lerner has an absorption mechanism in plasmas which produces the CMB by absorbing long wavelength radiation.  The same absorption problem arises from his model, insufficient absorption is seen at long wavelengths.)

Because of these two arguments, "fine tuning of the size-distribution" and "the necessary absorption to produce a blackbody spectrum is not observed", the hypothesis ***"dust causes the CMB" is falsified***.

[^1]: "Reducing the Uncertainty on the Dispersion Measure of FRB hosts," L. Marmet, FRB 2021 online Conference, Poster & Lightning talk [ID PL04](https://drive.google.com/file/d/1EC31IcpNJYum60DbxJp9exshWeJAD_5w/view?pli=1) (July 28th – August 5th, 2021)

**Reply by LifeIsStrange** [reply]
@RedshiftDrift 
Thanks for the answer Louis, I have a lot of respect for you but I wonder if you might not be overconfident on considering the CMB as dust, as clearly falsified.

To be clear, I am in between a layman and an expert, as I have read a very large number of papers in cosmology, yet I lack proper education in mathematics and here, in dust physics so I will say many things that are wrong and ask basic questions, this should still help clarify the debate.

Your update on electron density via FRB is interesting.
Regarding that topic, the paper from New Tired Light shows that there is a huge variance in the estimate of el depending on the analytic method, 
https://iopscience.iop.org/article/10.1088/1742-6596/1251/1/012007/pdf
Using the angular distance he gets 0.498 while
using the luminosity distance he gets 0.350
from the same FRB
Note that a way to considerably reduce the scatter would be to measure the angular distance via a cherenkov telescope, which is one order of magnitude more accurate IIRC
https://arxiv.org/abs/1904.06324
Note that a paper with 14 FRBs shows an el of 0.5 so the inter-agreement is low (but maybe there are methodological differences)
https://www.semanticscholar.org/paper/Data-from-14%2C577-cosmological-objects-and-14-FRBs-a-Ashmore/c4b7c6fdd7c1a54e86c34f82fc5cb71c0d4d04f1 (I assume they are located/have measured redshift)


> Dust models of the CMB have been proposed a long time ago (dust grains, carbon or metallic whiskers [F. Hoyle et al.], etc.). There are two major objections to those:

> Dust grains need to have a very specific size-distribution to produce a blackbody spectrum. Trinchera avoids mentioning this in his paper. There is no way to explain how dust grains would be formed with the exact size-distribution required to produce the CMB at 2.7 K, the required fine-tuning cannot happen naturally.

1) What is the size-distribution curve required, how complex is it? It is essential you and I have the same understanding of it.
Note that it is multiparametric, as like in LCDM where a free parameter can be altered to resolve a tension of another, if the electron density, and or, the dust size ratio and/or the dust spatial distribution and/or the mean dust density and or the dust chemical composition are altered then a specific size distribution curve can be relaxed to multiple possible ones, meaning there are multiple possible configurations.

2) To what extent is this really unlikely/fine tuned? You cannot falsify an idea without a quantitative assessment.

3) If it were to be proven as extremely unlikely (which you haven't done here), the proof should be resiliant to other systematics/biases

4) It should be compared to other known likelihoods of "accepted" fine tunings/coincidence problems
such as the dark energy coincidence problem and many other extreme fine tuning problems in LCDM or with the probability that the full moon perfectly align with the perceived solar radius as a creationist argument.

I listed in 1) every parameters of the equation to reproduce a black body, while there are some uncertainties with all of them, the uncertainty range perfectly allows CMB as dust radiation (assuming Trinchera "simple" equations are correct). IMO there is considerably less empirical pressure on constraining the CMB as dust radiation, than there is for inflation (single field) models that are almost entirely ruled out by BICEP3 CMB study and yet are still part of the "standard" model. I agree this is whataboutism, but regardless in absolute terms, the parameters of CMB as dust are not at all empirically falsified.
More than their uncertainty ranges, their preffered current values (at least at the time of the paper) perfectly agrees with Trinchera calculation, he cites studies for every parameters values, including empirical size of retrieved cosmic dust in the solar system, and chemical abundance and dust density.
Hence the paper argue that there is a **reverse coincidence** problem, the fact that his calculations works so well with the measured data is highly unlikely to happen by chance, it seems (given the low flexibility of every parameters). This is IMO a major underlooked argument, of all plausible configurations, we observe we are in one of the very few that are seemingly compatible with CMB as dust radiation.
> It is not a coincidence that the diameter of the targets and their densities match,
_respectively_, the order of magnitude (μm), exact size (10-1000 μm) as well as the chemical density of
the cosmic dust grains observed in space (on average 3200 Kg/m3). Therefore, the cosmic dust really
represents our researched space matter that emits a thermal radiation in equilibrium with the redshifted
starlight.

> 1) What is the size-distribution curve required, how complex is it? It is essential you and I have the same understanding of it.

The size distribution that generates a black body for a given solar surface temperature is given in figure 7.
Contrary to the idea that those curves would be highly complex and fails Occam's razor, I find this curve to be quite basic/simple.
Moreover, as he shows, since Dwarf stars (M class) are dominant, then only a small subset of the distribution curve matters 
> cosmic dust varies in the range 10-1000 μm

My understanding of the paper is sadly quite limited but:
1) an argument for finetuning/against cmb as dust radiation would be that either the mean density or size distribution would need to change/evolve with distance. My limited understanding seems to imply that his model allows a stable mean size distribution and density and is resilient to mild spatial variations, hence _if so_ no finetuning problem
2) another ad-hocness would be that there would be a _very_ sensitive ratio of each dust sizes to match a black body from a dwarf star.
My limited understanding seems to imply that any value within the range 10-1000 um would do and that the function is resilient to variations in said ratio. I am not sure wether this is true (might be related to the multiple lines in figure 7 (unclear)) but the paper seems to imply that the ranges and variations within range are permissive, _if so_ then again no finetuning problem, quite the contrary

Hence I see no finetuning issues however:

> average density of the dust grains ~= 4366 kg/m3

This density of dust grains seems _intuitively_ very high ? I don't understand how this is possible however he does cite an empirical study.

> There is no way to explain how dust grains would be formed with the exact size-distribution required to produce the CMB at 2.7 K

Since the size range is already found empirically it is not a question of wether they are formed, the question is are they formed in sufficent quantity. I suppose in an old and ionized universe it makes sense that cosmic dust would become extremely small, hence his range.

> Dust produces absorption, especially at shorter wavelengths. This absorption could be observed as a reddening of the spectrum (not "redshifting", but "reddening" like the red sunset). On the other hand, spectra of supernovae are extremely well characterized up to very large redshifts, and very little reddening is observed [Adam Riess' papers]. Intergalactic space is extremely transparent. This is incompatible with the required absorption needed to produce blackbody radiation.
(Eric Lerner has an absorption mechanism in plasmas which produces the CMB by absorbing long wavelength radiation. The same absorption problem arises from his model, insufficient absorption is seen at long wavelengths.)

This argument while interesting, is not quantified here for his size range.
IIRC ~50% of the baryonic mass is found within the IGM/ISM AKA not in stars. The idea of a warm, ionized IGM/ISM has been recently empirically validated IIRC, including via the dispersion measure of FRBs. There is also ample evidence that a large number of galaxies are partially or even fully obscured.
An example of massive extinction, IIRC, is the [zone of avoidance](https://en.wikipedia.org/wiki/Zone_of_Avoidance)
Another major effect as evidence of obscuration is the Gunn–Peterson trough effect.

as for reddening, there are many papers showing a reddening effect,
https://arxiv.org/abs/1606.03452
This one seems to shows an interesting correlation albeit they says
> , we conclude that photoelectric absorption, rather than dust attenuation, dominates the depletion of ionizing photons
https://arxiv.org/abs/astro-ph/0406293 shows dust reddening in quasars

as for obscuration/transparency
https://www.semanticscholar.org/paper/Disappearing-Galaxies%3A-The-Orientation-Dependence-Cochrane-Angl'es-Alc'azar/7199b85b7d135ff27c74cb7c449b75393eb80605
> Galaxies that are invisible in deep optical–near-infrared imaging but detected at longer wavelengths have been the focus of several recent observational studies, with speculation that they could constitute a substantial missing population and **even dominate** the cosmic star formation rate density at z ≳ 4.
see also
https://www.semanticscholar.org/paper/The-failure-of-testing-for-cosmic-opacity-via-the-Vavry%C4%8Duk-Kroupa/a3f43957d846bac59bcf81a528b34f6e92d71db3

note that there seems to be no consensual specific dust attenuation curve
> https://www.aanda.org/articles/aa/pdf/2023/06/aa45818-22.pdf

> All We Are Is Dust in the WIM

https://www.semanticscholar.org/paper/All-We-Are-Is-Dust-in-the-WIM%3A-Constraints-on-Dust-West-Gaensler/a6915294e33a821a93e14ace492d211409b82c98
paper that seems to shows high ionized dust abundance in our ISM.
This recent paper shows a temperature of 20−2+3 K in the ionized ISM which seems consistent with the temperature range near stars as shown in the cmb paper (figure 10).
They also measure local dust opacity τ 353/N H (WNMH I) = (1.0 ± 0.1) × 10−26 cm2
I have no idea if this could be used to confirm or rule out the CMB as dust radiation model.

To summarize my limited understanding, 
the values of the model are not empirically refuted
there is a reverse coincidence problem
the curves and ranges do not seems to require implausibility/fine tuning
the temperature evolution with distance seems unclear, I don't understand from his figure ten wether 2.7k is reached only at 5 light years or wether once this distance is reached, there is a long term 2.7k radiation that will propagate over long distances.
My limited understading of his model is that it is ISM based, an IGM based dust model or oort cloud model would have different constraints? An IGM one, covering much larger distances might be more resilient to opacity constraints?
The mean dust density seems to me inplausibly high but he cites a study and it is just an intuition, not a falsification.
edit as for dust density I believe I misunderstood the paper, he talks about chemical density hence no paradox however since he doesnt mention the actual average space dust density, I don't understand how his calculus works (cf equation 117 but thats more a me problem than a refutation of his paper)
The reddening and opacity "observed" levels might be used to refute his model maybe, however you have not done a quantitative analysis
.
I also believe there is a major **double counting** problem, meaning that, according to my limited understanding, in any interpretation of the CMB, including in LCDM, it is a fact that the IGM and ISM are dust and electron rich so even if for some reason, the parameters would not allow to reproduce a perfect 2.7K radiation, in any model, I believe, the dust still exists and since carbon is abundant it should generate a CMB background of a different temperature (for example of 5 kelvins). Now wouldn't this different, unavoidable dust microwave background of a wrong temperature, makes the "primordial" 2.7k background spectroscopically invisible? Wouldn't we observe such dust background and wouldn't it makes seeing the CMB 2.7K impossible?
I am not much qualified in spectroscopy to maybe there are or maybe there aren't ways to denoise this signal to noise ratio.
Secondly independently of that, is such a dust background observed? Wouldn't its absence necessarilly stipulate that the CMB background we observed necessarilly comes from dust ? otherwise what happen to the "missing" dust radiation?
This highly intriguing paper is related to my argument (albeit it talks about CMB massive distortion from dust obscuration while I talk about distortion from dust rethermalization (which they briefly mention))
https://arxiv.org/abs/1706.04771
> the CMB can also be distorted due to absorption by dust
producing a decline of the CMB intensity at all frequencies
This distortion should be high enough to be observable in
the CMB measurements. The maximum spectral distortion
of the CMB light coming from z = 10 is predicted at 300 GHz
being at least 20 times higher than the detection level of the
COBE/FIRAS measurements (Fixsen et al. 1996) and at
least **35 times higher** than the detection level of the WMAP
or Planck measurements (Hinshaw et al. 2009; Planck Col-
laboration et al. 2014). The CMB should be distorted also
in a perfectly transparent universe just due to absorption by
dust in galaxies. This effect is about one order lower than
that for the intergalactic opacity, but still above the detec-
tion level of the current CMB measurements.

The signal is 35 times above the sensitivity of our instruments and ~hundred of times (?) above the sensitivity of the incoming CMB studies in the next few years (PRISM, CMBS4, liteBIRD, BICEP3 and Simons).
Note there are probes dedicated to CMB distortions (see section 4.3)
https://arxiv.org/pdf/2307.08513 (APSERa, TMS, BISOU, COSMO, PIXIE)
(related BLAST, FYST, LSPE, PILOT, TIME)
and (CONCERTO, EXCLAIM, DESHIMA, OLIMPO)

In my opinion it would be an enormous mistake to ignore this, which is direct robust empirical evidence of the CMB as being non primordial (and consistent with dust generation).
Not only that but there are only 3 possible conclusions from his result:
1) That the universe is extremely transparent and that the IGM is almost free of dust.
Which is highly unlikely and even then he shows that ISM dust would makes the distortion detectable by our current instruments and clearly ruled out by out future instruments (in the next few years), hence 1) is not only highly unlikely but will soon be falsified
2) That the CMB is regenerated over distance (either by dust rethermalization or via another mechanism (synchrotrons, tired light, etc) to compensate for the dust obscuration.
3) That the CMB is a nearby source (e.g. in the solar system or in the local group)

related to 1) https://www.semanticscholar.org/paper/a3f43957d846bac59bcf81a528b34f6e92d71db3
note that the question of double counting/signal also apply to cmb polarization

**Reply by budrap00** [reply]
>average density of the dust grains ~= 4366 kg/m3
This density of dust grains seems intuitively very high ? I don't understand how this is possible however he does cite an empirical study.

I would have written that number off as a typo but I seem to remember other similar claims made here recently. That number isn't "very high" it is absurd. A cubic meter of dry sand has a density of @ $1600 kg/m^3$. Absurd isn't a strong enough word. I am curious though, was the cited "empirical study" an actual peer-reviewed paper?

https://www.gigacalculator.com/calculators/sand-calculator.php

**Reply by LifeIsStrange** [reply]
Thanks for the comparison it indeed does seems absurd and no it isn't a typo (as it is said with different values)
iirc the main component from the cosmic dust is magnetite? it might have significantly higher mass density than sand? Still is likely off by order of magnitudes. Very curious.
Edit:
I believe it is me that is completely misunderstanding what he means by target density but I am not sure.
I believe that by target density he wants to assess the mass/density of a single dust grain
 >  **chemical** density of the cosmic dust grains observed in space (on average 3200 Kg/m3)

You can see table 2 he shows a target density of 5180kg/m of Fe3O4 and in addition of 3580kg/m for magnesia.
I understand it as, supposing an hypothetical block of 1 cubic meter entirely made of cosmic dust (like your calculator), then _if_ cosmic dust was e.g. made entirely (100%) of Fe3O4 then the density of this hypothetical block would be 5180kg.
This only shows that Fe3O4 is ~3 times chemically denser than sand and says nothing about the mean cosmic dust density in space.
Quite clearly as he shows below the table 2, by density he aims to assess the mass of a single dust grain of a given size, by modelizing it as an hypothetical full cubic meter block and by taking into account the estimated ratios of chemical elements.
While that seems very clear, what about the mean dust density (not a single grain) in space? The terminology he use is ambiguous and that parameter seems ignored?
Moreover he has a target density relative to the star brightness, is he implicitly making the point that for a given distance (unspecified, maybe his 5 light years figure 10), assuming a real density X (unspecified), given an homogeneous distribution of dusts grains of size Y with _chemical cubic density_ Z, then after the calculated distance, a black body radiation would be generated isotropically.
> The average density obtained, according to equation (117), corresponds to a diameter
of 1075um

Well in my opinion it was simply a misunderstanding from my lack of education in mathematics but it would be nice to give me a proper explanation of what he means.
IMO the crux to understand what he means is the equation 117.

Note that he explicitly says that:
> ρ is the density of matter in the Universe. 

Then he says

> We have also to stress that the calculation method applied
allows to determine a diameter of the targets independently on Einstein’s density of the Universe ρ,
whose value is **not known** 

If we understand p as a proxy to says that the mean dust density in space is not known, he does manages to prove his model without this information. I have absolutely no idea how he manages to do that.

As for the study he cites
> It is compatible with the range of diameter detected by a study [41] which measured a grain density up
to 3200 Kg/m3

https://onlinelibrary.wiley.com/doi/epdf/10.1111/maps.12325
It is likely talking about _chemical_ density

an estimate of the real density from another paper
is
> Σdust is the dust mass
surface density. We derive Σdust ∼ 500 M pc−2 assuming Re ∼
1.2 kpc (Section 3.1) where τ ∼ 1 at ∼ 170 μm, suggesting that
the dust emission is optically thick up to FIR wavelengths

Not converted to g/m3 though and at z=4 for a specific galaxy

https://www.researchgate.net/profile/David-Elbaz-2/publication/339164637_Deceptively_cold_dust_in_the_massive_starburst_galaxy_GN20_at_zsim4/links/5f0f086d45851512999b2706/Deceptively-cold-dust-in-the-massive-starburst-galaxy-GN20-at-zsim4.pdf

**Reply by RedshiftDrift** [reply]
@LifeIsStrange 
>  I wonder if you might not be overconfident on considering the CMB as dust, as clearly falsified.
> 2. To what extent is this really unlikely/fine tuned? You cannot falsify an idea without a quantitative assessment.

Here are two quantitative assessments.

1. See cartoon below.  Each dust grain-size produces an intensity distribution as a function of wavelength.  The amount of dust in each band (colour coded, lower graph) must have a specific amplitude to reproduce the shape of the blackbody spectrum (top graph).

  The COBE/Planck probes have confirmed that the CMB is a blackbody spectrum for a large number of wavelength bands (some 40 bands as shown on the graph [https://en.wikipedia.org/wiki/Cosmic_microwave_background#Features](https://en.wikipedia.org/wiki/Cosmic_microwave_background#Features)) For this assessment, I assume only 8 bands have been confirmed accurately.  The exact shape of the spectrum generated by each dust size is not important, I am just counting the number of free parameters that need to have a specific value.

  The Planck/COBE temperature measurements are accurate to $18 \ \mu \mbox{K}$ out of $2.7 \mbox{ K}$, or 0.0007 %.  From Stefan-Boltzmann's law, $E = \sigma \times T^4$, this makes the uncertainty on the **intensity** of the spectrum four times larger, or **0.003 %**.

  Therefore, the amount of dust *in each band* must match a specific value to within **0.003 %** to reproduce the measured **intensity** of the CMB spectrum.  From band to band, the variation of the dust density is only roughly 10%, so the required fine tuning for each band has a probability of **0.003 %/10 % = 0.0003**.  Given that the first band defines the 2.7 K temperature, there are 7 more bands to finely tune (i.e. seven parameters to adjust) with a total probability of $0.0003^7 = 2 \times 10^{-25}$.

  I am therefore >99.99999999999999999999998% confident that this falsifies "CMB from dust".
I don't call that "overconfident"!

2. From a theoretical point of view, the blackbody spectrum is the result of an interaction of light with a medium that must interact with every single photon, absorb it and re-emit it.  This means that the medium appears black, hence the historical reason for the name "BLACKbody radiation".  (See any explanation for blackbody radiation: ***total absorption is a necessary condition***, e.g. ["A black body is an idealized object that absorbs all electromagnetic radiation it comes in contact with. It then emits thermal radiation in a continuous spectrum according to its temperature."](https://lco.global/spacebook/light/black-body-radiation/))

  By contrast, intergalactic dust is very transparent, even at the wavelengths covered by the CMB.  Radio telescopes observe a multitude of galaxies at these frequencies (e.g. [Extremely deep 150 MHz source counts from the LoTSS Deep Fields](https://www.aanda.org/articles/aa/full_html/2021/04/aa39998-20/aa39998-20.html)) **through all the dust present in the IGM**.  These galaxies are observed with 100% confidence.  Therefore, dust *cannot produce a blackbody spectrum because it does not have the necessary absorption required of a blackbody*.

  Unless another theory of blackbody radiation is proposed, "CMB from dust" is falsified with 100% confidence (none of the papers you cite gives an alternative theory to explain the blackbody spectrum).

  The observation of galaxies through dust at CMB frequencies falsifies "CMB from dust" at a confidence level of 100%.
I don't call that "overconfident"!

![no-dust](https://github.com/a-cosmology-group/acg/assets/16748402/5e0e5cdd-a66b-491d-8f91-bbb7a4d95e2b)

**Comment by ExpEarth** [comment]
`100% of studies shows no signal in gamma ray bursts`

That's no longer true.  There is at least this 2021 study showing it:

https://arxiv.org/abs/2108.00395

**Reply by LifeIsStrange** [reply]
True however 
> geometric mean-based binned data is consistent with the cosmological time dilation signature, whereas the analyses using unbinned durations show a very large scatter.

Hence the signal to noise ratio seems unconclusive
edit:
aslo very speculatively:
> _proper motion_ will induce a directionally-dependent time dilation over the sky.

from https://www.prophy.science/article/167680543/
Assuming this premise:
I just stumbled upon this paper
"Transient Events in the Circumnuclear Regions of AGNs and Quasars As Sources of Imitations of Proper Motions"
It might be that quasars have a proper motion illusion systematic but I'm not sure what would make it directionally dependent, maybe a magnetic dipole effect would be possible though. Might be related to the cmb dipole/dipole cosmology (very highly speculative)
 See also the related  VIM effect.
https://www.srg.cosmos.ru/publications/143/show

edit time dilation is also recontextualized in this paper:
https://iopscience.iop.org/article/10.1088/1742-6596/1251/1/012007/pdf
at section 7.3 Supernovae light curve broadening
He does not just cite crawford but explain it could also be a result of malmquist bias and moreover ask the question:
> One must ask, “have the papers purportedly showing ‘time dilation’ stood the test of time?” The
‘UNION 2.1 SNIA Compilation’ [29] lists all supernovae currently classed as SN Ia’s. Out of the 833
SNe Ia’s listed in the data sets only 580 passes ‘usability’ tests and are still included
